### PR TITLE
Refactor Landing screen and add analytics for buttons

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Most of these changes are refactors to change _buildXYZ methods into XYZ widgets. The only functional changes are adding analytics to the buttons on the landing screen - I'll add comments to those.

### CODE REVIEW

File: packages/devtools_app/lib/src/landing_screen.dart [hunk 1]
```diff
@@ -208,10 +167,12 @@ class _LandingScreenBodyState extends State<LandingScreenBody>
   }
 
   Future<void> _connectHelper() async {
+    ga.select(
+      analytics_constants.landingScreen,
+      analytics_constants.connectToApp,
+    );
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/landing_screen.dart:173]: new code

File: packages/devtools_app/lib/src/landing_screen.dart [hunk 2]
```diff
@@ -246,3 +205,88 @@ class _LandingScreenBodyState extends State<LandingScreenBody>
...
+        ],
+      ),
+    );
+  }
+
+  Future<void> _importFile(BuildContext context) async {
+    ga.select(
+      analytics_constants.landingScreen,
+      analytics_constants.importFile,
+    );
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/landing_screen.dart:246]: new code

File: packages/devtools_app/lib/src/landing_screen.dart [hunk 3]
```diff
@@ -246,3 +205,88 @@ class _LandingScreenBodyState extends State<LandingScreenBody>
...
+        ],
+      ),
+    );
+  }
+
+  void _onOpenAppSizeToolSelected(BuildContext context) {
+    ga.select(
+      analytics_constants.landingScreen,
+      analytics_constants.openAppSizeTool,
+    );
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/landing_screen.dart:289]: new code

**Review by jacob314 — APPROVED** [review]
lgtm

**Comment by kenzieschmoll** [comment]
Work towards #3270

**Pull request merged**